modules/api/static/schema-v0.0.4-dev.yml

```
@@ -44,6 +44,18 @@ paths:
       responses:
         '204':
           description: The visit stage was updated successfully
+  /sites:
+    get:
+      tags:
+        - Sites
+      summary: List all sites
+      responses:
+        '200':
+          description: A list of all sites; regardless of projects and/or user's permissions.
```

Should this be updated to reflect that there is a [userSiteMatch](https://github.com/aces/Loris/pull/7771/files#diff-53f7d851ca4412ecf8f287a9c194d02e267ef9e2705f7923a1f458308e565567R113) filter ?
```suggestion
          description: A list of all sites for the requesting user.
```